### ISSUE TITLE
Multi-select cell type?

### ISSUE THREAD

**Issue by hansgru** [handsontable/handsontable#2498]
Is there any "multi-select" or "multi-dropdown" or "multi-autocomplete" option for the cell types?

Is it possible to achieve this at all with handsontable?

Thank you.

**Comment by AMBudnik** [comment]
There is no functionality like this yet, but it is a good idea.

**Comment by hansgru** [comment]
The sad state is that at least now I found **no** usable editable web grid component (in a commercial or open source framework) capable of achieving multi-select :( (and I tried quite a few :) ), even if this requirement comes quite often.

I know that mostly in a single cell, there's not enough place for this, but in most required scenarios the users want only a few values to render in the cell.

Also in the case of selects (multi or not), another aspect is the "key-label" problem: the grid needs to render the label, but needs to get/set/send/receive the "key".
